src/huggingface_hub/fastai_utils.py

```diff
@@ -226,10 +227,14 @@ def _create_model_card(repo_dir: Path):
             Directory where model card is created.
     """
     readme_path = repo_dir / "README.md"
-
+    metadata["library_name"] = "fastai"
+    model_card = "---\n"
+    model_card += yaml.dump(metadata, default_flow_style=False)
+    model_card += "---\n\n"
+    model_card += README_TEMPLATE
```

Comment:
Maybe this could use the `modelcards` utils now that we have them. WDYT?

Here is a guide on it https://huggingface.co/docs/huggingface_hub/main/en/how-to-model-cards#from-text

Reply:
I thought of this as a short term fix but it makes sense to use modelcards. I'll modify the PR.